app/javascript/widget/helpers/widgetConfig.js

```diff
@@ -0,0 +1,64 @@
+export const getChannelConfig = () => window.chatwootWebChannel;
+
+export const useInboxAvatarForBot = () => {
+  return getChannelConfig().enabledFeatures.includes(
+    'use_inbox_avatar_for_bot'
+  );
+};
+
+export const hasAConnectedAgentBot = () => {
+  return !!getChannelConfig().hasAConnectedAgentBot;
+};
+
+export const getInboxAvatarUrl = () => {
+  return getChannelConfig().avatarUrl;
+};
+
+export const hasEmojiPickerEnabled = () => {
+  return getChannelConfig().enabledFeatures.includes('emoji_picker');
+};
+
+export const hasAttachmentsEnabled = () => {
+  return getChannelConfig().enabledFeatures.includes('attachments');
+};
+
+export const hasEndConversationEnabled = () => {
+  return getChannelConfig().enabledFeatures.includes('end_conversation');
+};
+
+export const isPreChatFormEnabled = () => {
+  return getChannelConfig().preChatFormEnabled;
+};
+
+export const getWebsiteName = () => {
+  return getChannelConfig().websiteName;
+};
+
+export const workingHoursEnabled = () => {
+  return getChannelConfig().workingHoursEnabled;
+};
+
+export const getWelcomeTitle = () => {
+  return getChannelConfig().welcomeTitle;
+};
+
+export const getWelcomeTagline = () => {
```

Comment:
Where is this being used?

Reply:
Forgot to remove it, realised later getChannelConfig() has a lot of options, exporting each one of them was not correct